### DISCUSSION TITLE
Delete books / editions of books?

### DISCUSSION THREAD

**Discussion by zeitwidrig** [bookwyrm-social/bookwyrm#2601] [Q&A]
I prematurely added a new book instead of adding an edition to an existing one.
Is there any way to delete such an entry? The only thing I can do is editing the book. Or maybe merging it as an edition to another book?

If it's any help, this is the entry in question: https://bookwyrm.social/book/813977/s/test-pls-ignore

**Comment by jaschaurbach** [comment]
There is no way to delete a book from the frontend at the moment. There are some Issues regarding book management (e.g. #2587, #2441, #2266 and others)

The only way would be to delete it direct from the database
